src/Illuminate/Foundation/Validation/ValidatesRequests.php

```diff
@@ -28,9 +28,13 @@
      *
      * @throws \Illuminate\Foundation\Validation\ValidationException
      */
-    public function validate(Request $request, array $rules, array $messages = [], array $customAttributes = [])
+    public function validate($request, array $rules, array $messages = [], array $customAttributes = [])
```

Comment:
How about `$data`? That's the variable name used in `Validator::make($data)` https://github.com/laravel/framework/blob/5.1/src/Illuminate/Validation/Factory.php#L91